microprofile-health/src/test/java/org/jboss/eap/qe/microprofile/health/HealthUtil.java

```diff
@@ -0,0 +1,34 @@
+package org.jboss.eap.qe.microprofile.health;
+
+import org.jboss.eap.qe.microprofile.tooling.server.configuration.ConfigurationException;
+import org.jboss.eap.qe.microprofile.tooling.server.configuration.arquillian.ArquillianContainerProperties;
+import org.jboss.eap.qe.microprofile.tooling.server.configuration.arquillian.ArquillianDescriptorWrapper;
+
+public class HealthUtil {
```

Comment:
`HealthUtil` sounds quite generic, could you rename to something like `HealthUrlProvider`, please?

Reply:
yep